test/typings/test-d/delete-query-builder.test-d.ts

```diff
@@ -38,4 +38,21 @@ async function testDelete(db: Kysely<Database>) {
     .orWhere('toy.price', '=', 0)
     .executeTakeFirstOrThrow()
   expectType<DeleteResult>(r5)
+
```

Comment:
let's add some test cases:

- `returningAll()` for single & multi-table.
- `returningAll([t1, t2])`.

Reply:
Added these tests.